### ISSUE TITLE
Unit 3 Training error

### ISSUE THREAD

**Issue by Rajan123xyz** [huggingface/deep-rl-class#149]
How do I correct the error below
!python train.py --algo dqn  --env SpaceInvadersNoFrameskip-v4  -f logs/
python3: can't open file 'train.py': [Errno 2] No such file or directory

**Comment by Rajan123xyz** [comment]
If this problem is on colab
try changing from
=>drive.mount('/content/gdrive')
%cd '/content/gdrive/'
to particular path where the train.py is present
=> drive.mount('content/gdrive')
%cd '/content/gdrive/My Drive/models/research/object_detection/'

**Comment by Rajan123xyz** [comment]
Even with that I still can't seem to correct it.  Cheers

**Comment by simoninithomas** [comment]
Hey there 👋 that's weird 🤔 did you follow every step of the colab? Especially the git clone part?

**Comment by Rajan123xyz** [comment]
Hi Thomas.  Thank you for the reply.
Yes, I did run the git clone. Let me try again tomorrow.

Cheers,
R

**Comment by Rajan123xyz** [comment]
Hi Thomas,

Just to keep you in the loop, I have pushed the model to Huggingface😀😎.
I think the issue was the runtime, when I rerun the cell the error seems to
resolve.  Thank you.

As per the course requirements, I just require a computer, internet access
and Huggingface account.  Will I require a GPU for later units.

Once again thank you. I'm looking to complete more than 90% of the course,
and get the certificate.

Happy holidays,
Raj Moodley

[image: image.png]

**Comment by simoninithomas** [comment]
Happy to hear that 🥳 don't hesitate to share the model on discord rl-i-made-this 
For the current units no. For later it's not mandatory but it can be good for the A2C and PPO since training on colab will be longer.

Happy holidays 🤗 I'm closing the issue
